modeltrans/fields.py

```diff
@@ -83,10 +82,20 @@ def contribute_to_class(self, cls, name):
     def db_type(self, connection):
         return None
 
+    def get_instance_fallback_chain(self, instance, language):
+        default = get_fallback_chain(language)
+
+        i18n_field = instance._meta.get_field("i18n")
+        if i18n_field.fallback_language_field:
+            custom_fallback = getattr(instance, i18n_field.fallback_language_field)
+            if custom_fallback:
+                return [custom_fallback] + [default]
+
+        return default
+
     def __get__(self, instance, instance_type=None):
-        # this method is apparantly called with instance=None from django.
-        # django-hstor raises AttributeError here, but that doesn't solve
-        # our problem.
+        # This method is apparantly called with instance=None from django.
+        # django-hstor raises AttributeError here, but that doesn't solve our problem.
```

Comment:
"our problem"? 

```suggestion
        """
        This method is called with instance=None from django.
        Django-hstor raises AttributeError here, but that doesn't solve our problem.
        """
```

Reply:
Would have been nice if I had been a little more elaborate on this back when I wrote it.